assets/js/modules/analytics/components/dashboard/DashboardUniqueVisitorsWidget.js

```diff
@@ -172,6 +130,57 @@ function DashboardUniqueVisitorsWidget( { WidgetReportZero, WidgetReportError }
 	);
 }
 
+/**
+ * Selects args used for sparkline report from the store.
+ *
+ * @since n.e.x.t
+ * @private
+ *
+ * @param {Function} select Registry select.
+ * @return {Object} Sparkline args.
+ */
+export const selectSparklineArgs = ( select ) => {
+	return {
+		metrics: [
+			{
+				expression: 'ga:users',
+				alias: 'Users',
+			},
+		],
+		dimensions: [ 'ga:date', 'ga:channelGrouping' ],
+		dimensionFilters: { 'ga:channelGrouping': 'Organic Search' },
+		url: select( CORE_SITE ).getCurrentEntityURL(),
+		...select( CORE_USER ).getDateRangeDates( {
+			offsetDays: DATE_RANGE_OFFSET,
+		} ),
+	};
+};
+
+/**
+ * Selects report args from the store.
+ *
+ * @since n.e.x.t
+ * @private
+ *
+ * @param {Function} select Registry select.
+ * @return {Object} Report args.
+ */
+export const selectReportArgs = ( select ) => {
+	return {
+		metrics: [
+			{
+				expression: 'ga:users',
+				alias: 'Total Users',
+			},
+		],
+		dimensions: [ 'ga:channelGrouping' ],
+		...select( CORE_USER ).getDateRangeDates( {
+			offsetDays: DATE_RANGE_OFFSET,
+			compare: true,
+		} ),
+	};
+};
+
```

Comment:
I agree with @eugene-manuilov, we shouldn't change this production code just to get the Storybook story setup simpler. Having exported functions in the same file as the component isn't good practice, so we shouldn't do that.

Let's provide the report options in Storybook manually the same way we do in the legacy versions of the story. This issue is not about changing _that_, it's only about migrating to the new proper way of CSF.